assemblies/assembly-enabling-authentication.adoc

```diff
@@ -61,3 +61,5 @@ include::assembly-authenticating-with-github.adoc[leveloffset=+1]
 
 include::assembly-authenticating-with-microsoft-azure.adoc[leveloffset=+1]
 
+
+include::modules/authentication/proc-enabling-authentication-with-your-custom-authentication-provider.adoc[leveloffset=+1]
```

Comment:
@gashcrumb Do you agree that it is something we want to document as part of the _Authentication_ title?

Reply:
Yes, I think so.  It's an alternate way of setting up authentication in RHDH, where our opinionated static authentication provider module is disabled.  This allows the user to install a suite of dynamic plugins that address their specific authentication concerns.

Reply:
Perfect.